packages/pyright-internal/src/analyzer/importResolver.ts

```diff
@@ -1161,6 +1162,18 @@ export class ImportResolver {
             importFound = resolvedPaths.length >= moduleDescriptor.nameParts.length;
         }
 
+        // Set priority based on if import is in the excluded list or not.
+        let priority = DefaultPriority;
+        if (importFound && sourceFilePath && !isNamespacePackage && resolvedPaths.length > 0) {
+            // Check the resolved path. If it's for an included file or the importing
+            // file is also excluded, treat as lower priority.
```

Comment:
Is the logic described in the comment backwards compared to the code? I'm assuming that a lower priority number means "lower priority" meaning that we'd push it further down the completion list.

It looks to me like the code is checking if the resolved path is included or the importing path is not, give it **higher** priority.

Reply:
Comment is backwards